components/hab/src/command/pkg/export/kubernetes.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
```

Comment:
There is a bit of annoying code duplication happening here as it is the same as `docker.rs` in this directory with basically `s/docker/kubernetes/` applied.

Reply:
Another small difference is with version - currently the kubernetes exporter follows the version of the Habitat operator. I'm not sure whether to go this way or to follow the versioning of hab CLI.